### PR TITLE
Updated Point SDK to 15.6.1

### PR DESCRIPTION
**Pull request by nehabluedot**
1. Enter `apiKey` and `accountToken` from pushio_config_debug.json in `AppDelegate.m`
2. Enter Bluedot `projectId` in `AppDelegate.m`
3. Build and run the App.
4. Init SDK and startGeoTrigger
5. Wait for a Zone Entry event, it will be followed by a Push generated by responsys

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs.
 To take the full advantage of the Oracle Cloud services, it is mandatory to integrate needed libraries/frameworks generally called as SDKs to your mobile app.
-At this moment, only iOS and Android mobile applications are supported by the SDK. 
+At this moment, only iOS and Android mobile applications are supported by the SDK.
 This page contains steps on how to update your mobile app’s source and integrate the app with Bluedot Point SDK.
 
 Please refer integration steps mentioned at : https://docs.bluedot.io/integrations/oracle-integration/
+
+For latest Bluedot Point SDK 15.6.1, New xcframework needs to be created as the PointSDK is built for iOS device and Simulators and it will report error as the OracleMinimalApp project only builds for iOS device.
```
Comment (jsprincep) [inline README.md]: The issue here is not that this app is only built for iOS devices, but that the Carthage dependency has not been linked correctly. If we follow the [Carthage instructions in our docs](https://docs.bluedot.io/ios-sdk/ios-quick-start/) the app builds correctly.

**Review by jsprincep — CHANGES_REQUESTED** [review]
According to the log output we need to enable the remote notification background mode for this app?

There's also a couple of files in this project which don't appear to do anything? `Empty.Swift` & `BDHelloPointSDK-BridgingHeader`.

File: README.md (hunk 1 above)
Comment (nehabluedot) [reply README.md]: Carthage instructions are working, we don't need to rebuild the xcframework.
Removed the extra steps from readme

File: BDHelloPointSDk.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -8,15 +8,14 @@
 
 /* Begin PBXBuildFile section */
 		78028E9B25DA47E7003A9BF4 /* PushIOManager.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 78028E9A25DA47E7003A9BF4 /* PushIOManager.framework */; };
+		9134A73927BCCEE900DA519C /* PushIOManager.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 9134A73827BCCEE900DA519C /* PushIOManager.framework */; };
```
Comment (jsprincep) [inline BDHelloPointSDk.xcodeproj/project.pbxproj]: We're doubling up on the PushIOManager framework - we should delete one

File: BDHelloPointSDk.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -93,9 +81,18 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		9134A73727BCCD5900DA519C /* Recovered References */ = {
```
Comment (jsprincep) [inline BDHelloPointSDk.xcodeproj/project.pbxproj]: There's nothing in the recovered references directory, delete it.

File: BDHelloPointSDk/Info.plist [hunk 4]
```diff
@@ -31,6 +31,8 @@
 	<key>UIBackgroundModes</key>
 	<array>
 		<string>location</string>
+		<string>processing</string>
```
Comment (jsprincep) [inline BDHelloPointSDk/Info.plist:34]: Why do we require background processing?

File: BDHelloPointSDk.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -29,30 +28,18 @@
...
-			runOnlyForDeploymentPostprocessing = 0;
-		};
-/* End PBXCopyFilesBuildPhase section */
-
 /* Begin PBXFileReference section */
 		78028E9A25DA47E7003A9BF4 /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9134A73827BCCEE900DA519C /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9134A73D27BCCF5C00DA519C /* BDPointSDK.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = BDPointSDK.framework; path = "Carthage/Checkouts/PointSDK-iOS/PointSDK/BDPointSDK.framework"; sourceTree = "<group>"; };
+		9160FED527BB2148001AEA99 /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9160FEDB27BC7C19001AEA99 /* BDPoint.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; path = BDPoint.xcframework; sourceTree = "<group>"; };
```
Comment (jsprincep) [inline BDHelloPointSDk.xcodeproj/project.pbxproj]: I think this is a leftover from the previous method of manually adding xcframework?

File: BDHelloPointSDk.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -29,30 +28,18 @@
...
-			);
-			name = "Embed Frameworks";
-			runOnlyForDeploymentPostprocessing = 0;
-		};
-/* End PBXCopyFilesBuildPhase section */
-
 /* Begin PBXFileReference section */
 		78028E9A25DA47E7003A9BF4 /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9134A73827BCCEE900DA519C /* PushIOManager.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = PushIOManager.framework; sourceTree = "<group>"; };
+		9134A73D27BCCF5C00DA519C /* BDPointSDK.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = BDPointSDK.framework; path = "Carthage/Checkouts/PointSDK-iOS/PointSDK/BDPointSDK.framework"; sourceTree = "<group>"; };
```
Comment (jsprincep) [inline BDHelloPointSDk.xcodeproj/project.pbxproj:32]: We're doubling up on the BDPointSDK framework - it's listed twice?

**Review by jsprincep — CHANGES_REQUESTED** [review]
Good to see it's working, but we need to clean it up before sharing it with the world 🙂

File: BDHelloPointSDk.xcodeproj/project.pbxproj (hunk 5 above)
Comment (nehabluedot) [reply BDHelloPointSDk.xcodeproj/project.pbxproj]: Yes, removed extra entries of xcframework

File: BDHelloPointSDk/Info.plist (hunk 4 above)
Comment (nehabluedot) [reply BDHelloPointSDk/Info.plist:34]: I think its required for receiving pushes while app is in background

File: README.md [hunk 7]
```diff
@@ -1,8 +1,8 @@
 # OracleMinimalApp-iOS
 Oracle Minimal App integration for iOS
-The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs. 
+The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of  Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs.
```
Comment (jsprincep) [inline README.md]: This README is still a little unclear on the need to finish integrating the Responsys SDK, let's bring it into alignment with the Android readme:

```suggestion

The Bluedot SDK integration with Oracle Responsys Mobile App Platform Cloud Service enables mobile apps to take advantage of the power of Oracle Cloud services and the superior accuracy, Geofence, Geolines™ and BLE Beacon triggering capabilities of the Bluedot SDKs.

### To add the Oracle SDK
Please follow detailed instructions at https://docs.oracle.com/en/cloud/saas/marketing/responsys-develop-mobile/android/android.htm

### To integrate Bluedot SDK
Please refer integration steps mentioned at https://docs.bluedot.io/integrations/oracle-integration/ios/
```

**Review by jsprincep — APPROVED** [review]
Pop in the README update, and we're looking good! 👌

File: README.md (hunk 7 above)
Comment (nehabluedot) [reply README.md]: Readme updated

**Pull request merged**